r2i/onnxrt/engine.cc

```diff
@@ -218,12 +280,12 @@ RuntimeError Engine::GetSessionInfo(std::shared_ptr<Ort::Session> session,
   Ort::AllocatorWithDefaultOptions output_allocator;
 
   try {
-    this->input_node_dims = this->GetSessionInputNodeDims(session, index);
-    this->output_size = this->GetSessionOutputSize(session, index);
+    this->input_node_dims = this->GetSessionInputNodeDims(session, index, error);
+    this->output_size = this->GetSessionOutputSize(session, index, error);
     this->input_node_names[index] = this->GetSessionInputName(session, index,
-                                    input_allocator);
+                                    input_allocator, error);
```

Comment:
Input allocator is not a pointer, but the method expects a pointer. How is this working?